definitions/marts/looker_studio_marts/npq/ls_npq_acceptance_reversal.sqlx

```diff
@@ -0,0 +1,63 @@
+config {
+    type: "table",
+    assertions: {
+        uniqueKey: ["application_ecf_id"]
+    },
+    bigquery: {
+        clusterBy: ["cohort", "provider_name", "short_course_name"]
+    },
+    description: "This mart produces a table of applications that have had at least one instance where the accepted approval status has been reversed to pending. \n \
+    The data produced from this mart allows for NPQ Contract Management to monitor the volume of this activity as well as what then happens to the approval status of the application via the current_application_status column. \n \
+    The volume of applications where this activity has occurred can be aggregated at cohort, lead provider, course and current application status levels.",
+    columns: {
+        current_application_status: "The current application status as per the service.",
+        accepted_reversal_count: "The number of times the application status of the application has been reverted from accepted to pending."
+    }
+}
+
+WITH acceptance_reversion AS (
+  SELECT
+    status_changes.entity_id AS application_ecf_id

```

Comment:
Apart from the `application_ecf_id`, is there any reason why you need to group by the other fields at this level? Could you not just group by at this id level?

Reply:
The group by all does two things in one go. It groups the count and also removes duplicated rows without the need of an additional qualify based on update occurred at date.